src/bootstrap/Cargo.toml

```diff
@@ -21,7 +21,7 @@ test = false
 
 [[bin]]
 name = "rustdoc"
```

Comment:
this needs to be `rustdoc-shim` in order to fix https://github.com/rust-lang/rust/issues/112082

Reply:
which requires updating https://github.com/rust-lang/rust/blob/a9baa16482ba4100df529ccba39c787f27ad0475/src/bootstrap/dist.rs#L2312-L2314 too